fempy/sim/LPi/SimLPi.C

```diff
@@ -0,0 +1,405 @@
+#include <array>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "yaml-cpp/yaml.h"
+
+#include <TFile.h>
+#include <TROOT.h>
+#include <TH1.h>
+#include <TH2.h>
+#include <TMath.h>
+#include <TDatabasePDG.h>
+#include <TRandom3.h>
+#include <TSystem.h>
+#include <TF1.h>
+#include <TLorentzVector.h>
+
+#include "Pythia8/Pythia.h"
+
+enum tunes {kMonash = 0, kCRMode0, kCRMode2, kCRMode3};
+
+using namespace Pythia8;
+
+float RelativePairMomentum(TLorentzVector &PartOne, TLorentzVector &PartTwo) {
+  TLorentzVector trackSum = PartOne + PartTwo;
+
+  float beta = trackSum.Beta();
+  float betax = beta * cos(trackSum.Phi()) * sin(trackSum.Theta());
+  float betay = beta * sin(trackSum.Phi()) * sin(trackSum.Theta());
+  float betaz = beta * cos(trackSum.Theta());
+
+  TLorentzVector PartOneCMS = PartOne;
+  TLorentzVector PartTwoCMS = PartTwo;
+
+  PartOneCMS.Boost(-betax, -betay, -betaz);
+  PartTwoCMS.Boost(-betax, -betay, -betaz);
+
+  TLorentzVector trackRelK = PartOneCMS - PartTwoCMS;
+
+  return 0.5 * trackRelK.P();
+}
+
+// Macro for files NewSimStandalonePythia
+void SimLPiBkg(int seed=42, int nEvents=250000, tunes = kCRMode2,
+               std::string MCFilePath="/home/mdicostanzo/an/LPi/Trains/02_allpc/mc/data/AnalysisResultsAllPC.root",
```

Comment:
Put the application of the momentum smearing as an option (I think it would be even better to keep the application of the momentum smearing in a separate macro but up to you)